website/docs/cli/commands/describe/describe-affected.md

```diff
@@ -0,0 +1,112 @@
+:::
+
+## Examples
+
+```shell
+atmos describe affected
+atmos describe affected --verbose=true
+atmos describe affected --ref refs/heads/main
+atmos describe affected --ref refs/heads/my-new-branch --verbose=true
+atmos describe affected --ref refs/heads/main --format json
+atmos describe affected --ref refs/tags/v1.16.0 --file affected.yaml --format yaml
+atmos describe affected --sha 3a5eafeab90426bd82bf5899896b28cc0bab3073 --file affected.json
+atmos describe affected --sha 3a5eafeab90426bd82bf5899896b28cc0bab3073
+```
+
+## Flags
+
+| Flag        | Description                                                                                                                                                     | Required |
+|:------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|:---------|
+| `--ref`     | Git reference with which to compare the current branch. Refer to [Git References](https://git-scm.com/book/en/v2/Git-Internals-Git-References) for more details | no       |
+| `--sha`     | Git commit SHA with which to compare the current branch                                                                                                         | no       |
+| `--file`    | If specified, write the result to the file                                                                                                                      | no       |
+| `--format`  | Specify the output format: `json` or `yaml` (`json` is default)                                                                                                 | no       |
+| `--verbose` | Print more detailed output when cloning and checking out the Git repository                                                                                     | no       |
+
+## Output
+
+The command outputs a list of objects (in JSON or YAML formats).
+
+Each object has the following schema:
+
+```json
+{
+  "stack": "....",
+  "component_type": "....",
+  "component": "....",
+  "affected": "....."
+}
+```
+
+where:
+
+- `stack` is the affected Atmos stack
+- `component` is the affected Atmos component in the stack
+- `component_type` is the type of the affected Atmos component (`terraform` or `helmfile`)
+- `affected` shows what was changed for the component. The possible values are:
+
+  - `vars` - the `vars` component section in the stack config has been modified
+  - `env` - the `env` component section in the stack config has been modified
+  - `settings` - the `settings` component section in the stack config has been modified
+  - `metadata` - the `metadata` component section in the stack config has been modified
+  - `terraform` - the Terraform component (Terraform files) that the affected Atmos component provisions has been changed
+  - `helmfile` - the Helmfile component (Helmfile files) that the affected Atmos component provisions has been changed
```

Comment:
IMO, this should just be consolidated to `component`, as `component_type` will be the indicator of if it's `terraform` or `helmfile`

Reply:
Consider changing affected to be disambiguated a little bit: `stack.vars` or `stack.env`.